docs/recipes/install/common/warewulf4_setup_centos.tex

```diff
@@ -0,0 +1,38 @@
+% begin_ohpc_run
+% ohpc_validation_newline
+% ohpc_validation_comment Update /etc/hosts template to have ${hostname}.localdomain as the first host entry
+% ohpc_command sed -e 's_\({{$node.Id.Get}}{{end}}\)_{{$node.Id.Get}}.localdomain \1_g' -i /srv/warewulf/overlays/host/rootfs/etc/hosts.ww
```

Comment:
Could you run a second command here:
```
bash /etc/profile.d/ssh_setup.sh
```
The script only runs if a new login happens and we don't do that during the run of the recipe. So if SSH is setup early that would help CI and our users.

Reply:
Adding - will commit in a few min, just committed a documentation update and missed this comment, expect a new commit.